src/style.css

```
@@ -1 +1,64 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+body,
+ul {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  padding: 23px 0;
+  display: flex;
+  text-transform: uppercase;
+  line-height: 14px;
+  color: #000;
+  text-decoration: none;
+}
+
+.logo {
```

```suggestion
.header__logo {
```
isn't this logo an element of header?